### DISCUSSION TITLE
Keep Hidra analysis?

### DISCUSSION THREAD

**Discussion by lopezzot** [DRCalo/DRTB23Sim#4] [Analysis] [answered]
Hi,
@polesell @EdoPro98 I see that the is some data analysis code under the `analysis/` folder. It is inherited from the proto-code for the Hidra simulation. Should we keep it within the 2023 test-beam code or delete it from this repo? Please let me know.

**Comment by polesell** [comment, marked as answer]
Ciao Lorenzo,

    ci ho gurdato, e' un pezzo di codice elementare per come accedere alle varibili,
    modulare per la geometria, direi che si puo' tenere come esempio.

    Sempre per l'analisi, ho recuperato la versione modificata del codice usato per
    produrre le ntuple del testbeam dai dati che prende come input l'uscita della
    simulazione, e sto lavorando per adattarla a hidrasim, e poi faccio il commit.

    Ciao,
                                                 Giacomo
________________________________
From: Lorenzo Pezzotti ***@***.***>
Sent: Friday, June 9, 2023 15:57
To: DRCalo/DRTB23Sim ***@***.***>
Cc: Giacomo Polesello ***@***.***>; Mention ***@***.***>
Subject: [DRCalo/DRTB23Sim] Keep Hidra analysis? (Discussion #4)


Hi,
@polesell<https://github.com/polesell> @EdoPro98<https://github.com/EdoPro98> I see that the is some data analysis code under the analysis/ folder. It is inherited from the proto-code for the Hidra simulation. Should we keep it within the 2023 test-beam code or delete it from this repo? Please let me know.

**Comment by lopezzot** [comment]
Hello @polesell,

ok, let's keep it as an example for the moment. I suggest using this part later on for the analysis by renaming or using it as a skeleton to add a new data analysis. So we avoid Hidra code here.

About the data converter. As far as I can tell now, the output format of the 2023 test-beam will be (almost) identical to the 2021 one. I do not remember if we decided to put it in the simulation or as a dedicated step after the simulation. I would probably prefer to have it directly in the simulation this time. Let me know if any help is needed with that.
